percona-monitoring-and-management/install.sh

```diff
@@ -1,5 +1,28 @@
+#!/bin/bash
+PMM_PASSWORD=$(echo ${PMM_ADMIN_PASSWORD}|base64)
+
+# Creating secret
+envsubst < secret.yml|kubectl apply -f -
+
+# Add the Percona Helm repository
 helm repo add percona https://percona.github.io/percona-helm-charts/
+
+# Update your local Helm chart repository cache
 helm repo update
-kubectl create ns pmm
 
-helm install pmm percona/pmm --namespace pmm  --set secret.pmm_password=$PMM_ADMIN_PASSWORD --set service.type=$SERVICE_TYPE --set-string pmmEnv.ENABLE_DBAAS="1"
+# Install the PMM  Helm chart
+helm install pmm percona/pmm --set secret.create=false --set service.type=$SERVICE_TYPE
+
+# Wait for pod to be ready
+kubectl wait pods --for condition=Ready --timeout=120s pmm-0
```

Comment:
https://github.com/percona/pmm-qa/blob/main/pmm-tests/pmm-2-0-bats-tests/k8s_helper.sh#L4

120s might not be enough, our container is 1.2Gb

Reply:
Okay. even better